jellyfin_kodi/helper/xmls.py

```diff
@@ -71,3 +71,64 @@ def advanced_settings():
             xbmc.executebuiltin('RestartApp')
 
             return True
+
+def verify_kodi_defaults():
+    ''' Make sure we have the kodi default folder in place.
+    '''
+
+    source_base_path = translate_path("special://xbmc/system/library/video")
+    dest_base_path = translate_path("special://profile/library/video")
+
+    if not os.path.exists(source_base_path):
+        LOG.error("XMLs source path `%s` not found.", source_base_path)
+        return False
+
+    # Make sure the files exist in the local profile.
+    # TODO: Investigate why this is needed.
+    # I would think Kodi pulls data from the default profile
+    # if we don't do this.
```

Comment:
This comment probably isn't necessary anymore, as we know it doesn't pull from the default.